apps/wdos/wdos.yml

```diff
@@ -0,0 +1,15 @@
+name: wdOS
+description: 'The easy way to make desktop apps with web languages and nothing else.'
+website: 'https://www.webfra.me/'
```

Comment:
This link leads to absolutely another application. If you can, please create a direct landing page, or point this link to the GitHub repository.

Reply:
I updated the URL and description to be more accurate. Essentially this is an electron app that allows people to add Nodejs or basic html sites to a folder in there desktop and be able to run this using this program as a shell. It also allows a person to spin up a express web server and serve there site as a website by adding it to a different folder.